library/ssl_tls.c

```diff
@@ -4751,10 +4751,16 @@ int mbedtls_ssl_conf_dtls_srtp_protection_profiles( mbedtls_ssl_config *conf,
     return( 0 );
 }
 
-const mbedtls_dtls_srtp_info *
+mbedtls_dtls_srtp_info
```

Comment:
Most Mbed TLS functions don't return structures, usually they have output parameters for returning structures. This way the caller owns the memory management of the result and can decide if they wish to store it on the stack or on the heap. By output parameter I mean that the signature of the function could be like this:
```c
void mbedtls_ssl_get_dtls_srtp_negotiation_result( const mbedtls_ssl_context *ssl, mbedtls_dtls_srtp_info *dtls_srtp_info )
```

Could we please have an output parameter for the result here?

Reply:
no problem, here it is.

Reply:
Thank you very much!